src/corebluetooth.cs

```diff
@@ -850,13 +854,9 @@ interface CBATTRequest {
 	// `delloc` a default instance crash applications and a default instance, without the ability to change the UUID, does not make sense
 	[DisableDefaultCtor]
 	interface CBCentral : NSCopying {
-#if MONOMAC
-		// Introduced with iOS7, but does not have NS_AVAILABLE
-		// Moved to a new base class, CBPeer, in iOS 8.
-		[iOS (7,0)]
```

Comment:
This iOS seems to have been a copy & paste issue, as we were in a MONOMAC block.